lib/src/schedule.dart

```diff
@@ -161,6 +167,9 @@ class Session implements Comparable<Session> {
 
   final String imageUrl;
 
+  bool get isDivider =>
+      imageUrl == null && description == null && presenters == null;
```

Comment:
Not sure if this is the best way to determine if a session is a divider, but it will have to do for now.

Reply:
Yeah, definitely something we can tweak, up to and including adding a book field in firebase.